src/refactor_nrepl/util.clj

```diff
@@ -52,3 +52,17 @@
     (string/replace-first _ match rep)
     (reverse _)
     (apply str _)))
+
+(defn inlined-dependency? [candidate]
+  (or (-> candidate str (.startsWith "deps."))
+      (-> candidate str (.startsWith "mranderson"))
+      (-> candidate str (.startsWith "eastwood.copieddeps"))))
+
+(defn- refactor-nrepl? [candidate]
+  ;; the refactor-nrepl implmentation should never analyse itself w/ tools.analyzer, as this can cause opaque issues:
+  (and (-> candidate str (.startsWith "refactor-nrepl"))
+       (not (-> candidate str (.contains "test")))))
+
+(def invalid-fqn?
```

Comment:
I don't like using `def` to create functions because I then lose the editor typehint and I use those a lot to remind me what sort of arg(s) are expected.

Maybe we can give this a name like `self-referential?` or something to that effect?  My thinking is that this isn't a generally invalid fqn, but invalid in the context of refactor nrepl.